pkg/okteto/auth.go

```diff
@@ -143,7 +145,7 @@ func queryUser(ctx context.Context, client *graphql.Client, token string) (*q, e
 	var user q
 	q := `query {
 		user {
-			id,name,email,externalID,token,new,registry,buildkit,certificate
+			id,name,email,externalID,token,new,registry,buildkit,certificate,globalnamespace
```

Comment:
If you run this against Cloud, it will fail, because we don't support this field yet.
We should catch the error in that case, and do the query without `globalnamespace`.
There is similar code when deploying a pipeline, for example.